src/extension.ts

```diff
@@ -35,7 +37,9 @@ function searchAll() {
     var query = getSelectedText(editor);
 
     var dash = new Dash();
-    exec(dash.getCommand(query));
+    var OS = platform();

```

Comment:
another improvement we can make is to move this line after line 4. So, when every `search()` or `searchAll()` is called, `platform()` won't being called again. 😃